monkey/monkey_island/cc/services/attack/technique_reports/T1003.py

```diff
@@ -0,0 +1,25 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.database import mongo
+
+__author__ = "VakarisZ"
+
+
+class T1003(AttackTechnique):
+
+    tech_id = "T1003"
+    unscanned_msg = "Monkey tried to obtain credentials from systems in the network but didn't find any or failed."
+    scanned_msg = "Monkey tried to obtain credentials from systems in the network but didn't find any or failed."
+    used_msg = "Monkey successfully obtained some credentials from systems on the network."
+
+    query = {'telem_type': 'system_info_collection', '$and': [{'data.credentials': {'$exists': True}},
+                                                              {'data.credentials': {'$gt': {}}}]}
```

Comment:
is this query even working?
Isn't there missing a number after '$gt'?

Reply:
It's working and It's the only way I found to check if object is not empty.

Reply:
That's fair enough, but because this looks weird, you should at least add a comment specifying that's what this syntax does